sympy/core/power.py

```diff
@@ -124,6 +124,15 @@ class Pow(Expr):
     | (-oo)**oo    | nan     |                                               |
     | (-oo)**-oo   |         |                                               |
     +--------------+---------+-----------------------------------------------+
+    | oo**I        | nan     |                                               |
+    | (-oo)**I     |         |                                               |
+    +------------------------------------------------------------------------+
+    | oo**(1+I)    | zoo     |                                               |
+    | (-oo)**(1+I) |         |                                               |
+    +------------------------------------------------------------------------+
+    | oo**(-1+I)   | 0       |                                               |
+    | (-oo)**(-1+I)|         |                                               |
```

Comment:
Going by [this](https://github.com/gxyd/sympy/blob/inf_run_time/sympy/core/power.py#L83) line, I think what you are doing is Ok.